### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by CollinsTatang**
Intensive work is done on this project. 
Waiting for your respond

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### ### INVALID 🛑 
Hello, @CollinsTatang nice progress. You have done such an impressive job 👍🏿 . However, it seems you forgot to add a readme to your project. Remember for your project to be reviewed it needs to meet all the minimum requirements. 

[Here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) is a link to the minimum requirements that must be met.

No worries just add a readme file and submit your project for another review. 

Note: I also noticed that you have some unresolved linter issues. Kindly resolve this too.

**if you have any question, kindly reached me on slack** @peterrobert

### HAPPY CODING 💻

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [x] **Descriptive PR**

1.	Add more descriptive information to your PR message. Please tell something interesting to the TSEs and what you liked about the project. 
2.	The bullet points are nice to add.

- [x] **Descripive Readme**
Please add a proper description in your Readme. [Here](https://github.com/arslanbisharat/New-York-Times-Page/blob/master/README.md) is an example. 

- [x] You have a number of linter errors. Please work on it. Always check if you have any errors before requiring a code review. 
- [x] Don't use IDs in selectors.
- [x] You are missing the bottom section(Suggested newsletters for you, Most Popular, Editors’ Picks) of the original webpage. Please add those sections. 
- [x] Everything in the navbar is actually clickable it's mean those are links somehow. Please work on it. 


- [ ]  Editors’ Picks section is missing on your webpage. Please add that section. 


- [ ] I can see your work with your partner on this project but you submitted this project individually. You should not click "I worked alone on this project" if the project is collaborative. For the next submission please avoid repeating it. 


**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🔧
### Hi There Team✋🏽
 
- [x] Nice work, your ReadMe file looks good but it can look even better. Using the example of the proposed standard by Microverse for the Readme in the last review you got, you can improve your readme file and make it professional by simply forking this [Readme Template](https://github.com/microverseinc/readme-template):

that way, you can add an author's section for example that will look like this:
author1
which looks more presentable.

- [x] Also please fix all linter errors too and then submit for another pull request.
- [ ] Your pull request description can be better too

Feel free to reach out to me on slack if you have any questions, Witah Georjane 🤝

Good luck on your project and Happy Coding 💻

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
- [ ] Lets fix linter issues
- [ ] Lets align the editors pick a little to the left


- [ ] Lets write a concise readme.MD file that contains project title, description and technologies used
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello guys**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] The project must have a descriptive README.md ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
- [ ] Please make sure to fix all of your linters errors.
- [ ] most of your images are not in tteh repo, please make sure to add them and fix all of the broken links.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [ ] As requested by every TSEs, Please remove linter errors before requesting Code Review. 


- [ ] As requested by every TSEs, Please add proper instructions in your Readme file.[This ](https://github.com/tta2yta/The-New-York-Times-WebSite/blob/feature/README.md)is how it should look like. 
- [ ] ⦁	Descriptive PR
1.	Add more descriptive information to your PR message. Please tell something interesting to the TSEs and what you liked about the project. 
2.	The bullet points are nice to add.

- [ ] These must be links in your Clone: 
1- Everything in the navbar (Buttons, hamburger, and search icon should be clickable while LOGO and SPACE & COSMOS should be links). 
2- LOGO in the Footer section. 
3- Ever item in the footer

- [ ] You are missing the most important sections of the webpage (Editor's pick on the side and More in Space and Astronomy and other associated sections). 
- [ ] Buttons in the navbar should have the same color as present on the original page. 


**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by ntzwilly** [comment]
Starting to make required changes

**Comment by ntzwilly** [comment]
Pull request description:

1. Add README.md file using Microverse's template
2. Remove Linter errors
3. Add the bottom section(Suggested newsletters for you, Most Popular, Editors’ Picks) of the original webpage.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1128 @@
...
+  <body>
+    <!-- Header -->
+
+    <nav id="header">
+      <div>
+        <img class="icon" src="images/bars-solid.svg" alt="bar" />
+        <img class="icon" src="images/search-solid.svg" alt="search icon" />
+        <a id="space" href="#"><b>SPACE & COSMOS</b></a>
+      </div>
+      <img id="logo" src="images/nyt.png" alt="header image" />
```
Comment (Spayco) [inline index.html:19]: - [ ] Logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1128 @@
...
+              Family Attic</a
+            >
+          </div>
+          <div class="pick">
+            <a href="#"
+              ><img src="images/Sorey.jpg" alt="" />The Composer Tyshawn Sorey
+              Enters a New Phase</a
+            >
+          </div>
+        </div>
```
Comment (Spayco) [inline index.html:104]: - [ ] you should use an  [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] 3 simple changes, check below :point_down:

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
...
+This project features HTML5 semantic tags, CSS float, CSS flexbox, and Grid.
+
+## Built With
+
+- HTML5 & CSS3
+
+## Live Demo
+
+[Live Demo Link]()
+
```
[the remark this answers is not in the document]
Comment (ntzwilly) [reply README.md]: A valid link to the GitHub Pages site is added to the README.md

File: index.html (hunk 1 above)
Comment (ntzwilly) [reply index.html]: The logo is now a link

File: index.html (hunk 2 above)
Comment (ntzwilly) [reply index.html]: Editor's picks section on the rightmost side use now semantic tag <aside> instead of <div>

**Comment by ntzwilly** [comment]
Pull request description:
1. README.md has a valid link to the live demo of the project.
2. The logo on the header section is now a link.
3. Semantic tag <aside> is used for editor's picks instead of div.

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:


_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request closed without merging**